custom/enikshay/reports/views.py

```diff
@@ -0,0 +1,45 @@
+from django.http.response import JsonResponse
+from django.utils.decorators import method_decorator
+from django.views.generic.base import View
+
+from corehq.apps.domain.decorators import login_and_domain_required
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.userreports.reports.filters.choice_providers import ChoiceQueryContext
+
+
+class LocationsView(View):
```

Comment:
We have a similar view for UCR filters but it takes as argument ```report_id``` so I copied and modified it a little bit.